README.md

```diff
@@ -10,54 +10,101 @@ Please note; this is the _INVERSE_ of some typical standalone project setup such
 
-This root level CDK app contains 4 major stacks: `stateful-pipeline`,`stateless-pipeline` , `stateful` and `stateless`. Pipeline stack is the CI/CD automation with CodePipeline setup. The `stateful` stack holds and manages some long-running AWS infrastructure resources. The `stateless` stack manages self-mutating CodePipeline reusable CDK Constructs for the [MicroService Applications](docs/developer/MICROSERVICE.md). In terms of CDK deployment point-of-view, the microservice application will be "stateless" application such that it will be changing/mutating over time; whereas "the data" its holds like PostgreSQL server infrastructure won't be changing that frequent. When updating "stateful" resources, there involves additional cares, steps and ops-procedures such as backing up database, downtime planning and so on; hence stateful. We use [configuration constants](./config) to decouple the reference between `stateful` and `stateless` AWS resources.
+There are 2 CDK apps here:
 
-In most cases, we deploy with automation across operational target environments or AWS accounts: `beta`, `gamma`, `prod`. For some particular purpose (such as onboarding procedure, isolated experimentation), we can spin up the whole infrastructure into some unique isolated AWS account. These key CDK entrypoints are documented in the following sections: Automation and Manual.
+- Stateful
+  
+  This holds and manages long-running AWS stateful resources. The resources will typically be something that won't be
+  changing frequently and could not be torn down easily. For example, the RDS Cluster which contains application data. When
+  updating "stateful" resources, additional care is needed such as backing up the database, downtime planning and so on;
+  hence stateful.
+
+- Stateless
+
+  As the opposite of stateful resources, stateless resources will have the ability to redeploy quickly without worrying
+  about any retainable data. For example, AWS lambdas and API Gateway have no retainable data when destroyed and spin up
+  easily. The [MicroService Applications](docs/developer/MICROSERVICE.md) resources will usually be here and they will
+  have a lookup from stateful resources when needed.
+
+You could access the CDK command for each app via `yarn cdk-stateless` or `yarn cdk-stateful`. The `cdk-*` is
+just a CDK alias that points to a specific app, so you could use `cdk` command natively for each app (e.g. `yarn cdk-stateless --help`).
+
+We use [configuration constants](./config) to reference constants between the `stateful` and `stateless` CDK apps.
+
+In most cases, we deploy with automation across operational target environments or AWS accounts: `beta` (dev), `gamma` (staging),
+`prod`. For some particular purpose (such as onboarding procedure, or isolated experimentation), we can spin up the
+whole infrastructure into some unique isolated AWS account.
```

Comment:
Would it be useful to mention how to spin up the whole infrastructure without relying on the specific account config?

Reply:
Yes, I think instruction to spin up from scratch would be useful. Although, I think we need to do some refactoring the config file? I feel that our config file is tied down to a specific account number

Reply:
Yes, that's a good point.

Reply:
#167